app/models/comment.rb

```diff
@@ -2,7 +2,11 @@ class Comment < ApplicationRecord
   belongs_to :post, class_name: 'Post', foreign_key: :post_id
```

Comment:
- [ Optional ] Please add a zero default value for the posts and comments and likes counters so we can add them to the database directly 
![image](https://user-images.githubusercontent.com/74744787/216772651-3cf97c8f-dfe7-484d-a37f-eda7c093f1b9.png)